pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -291,9 +291,27 @@ def load_data(input_file, ifos, rw_snr_threshold=None, data_tag=None,
         return None
 
     trigs = HFile(input_file, 'r')
-    rw_snr = trigs['network/reweighted_snr'][:]
-    net_ids = trigs['network/event_id'][:]
+    rw_snr = trigs['network/reweighted_snr'][:] \
+        if 'network/reweighted_snr' in trigs.keys() else numpy.array([])
+    net_ids = trigs['network/event_id'][:] \
+        if 'network/event_id' in trigs.keys() \
+        else numpy.array([], dtype=numpy.int64)
 
+    # Output the number of items loaded only upon a request by the user who is
+    # expected not to set data_tag to 'trigs'or 'injs' when processing the
+    # onsource
+    if data_tag == 'trigs':
+        logging.info("%d triggers loaded.", len(rw_snr))
+    elif data_tag == 'injs':
+        logging.info("%d injections loaded.", len(rw_snr))
+    else:
+        logging.info("Loading triggers.")
+    ifo_ids = {}
+    for ifo in ifos:
+        ifo_ids[ifo] = trigs[ifo+'/event_id'][:] \
+            if ifo+'/event_id' in trigs.keys() \
+            else numpy.array([], dtype=numpy.int64)
+    trigs.close()
     # Output the number of items loaded only upon a request by the user who is
```

Comment:
Lines 315 to 327 can be removed: they are a duplicate (older version) of the lines right above them.